### PR TITLE
make templates.rake Rails 4 compatible

### PR DESCRIPTION
**Pull request by mmoll**
Still works for me with Foreman 1.10. We need a 2.0.3 release then.

### CODE REVIEW

File: lib/templates.rake [hunk 1]
```diff
@@ -32,7 +32,7 @@ def update_template
     raise NoKindError
   end
 
-  db_template = ProvisioningTemplate.find_or_initialize_by_name(@name)
+  db_template = ProvisioningTemplate.where(name: @name).first_or_initialize
```
Comment (domcleal) [inline lib/templates.rake]: Nitpick, usually we use hash "rockets" style, `where(:name => @name)`

**Comment by domcleal** [comment]
Tests pass now, so that's good.  :+1:

**Comment by GregSutcliffe** [comment]
Merged as 92533afd0139261950199811ed0e5626c028a8a5, fixed the nitpick (hope you don't mind @mmoll :P). Thanks!

**Comment by mmoll** [comment]
@GregSutcliffe not at all :)

**Pull request closed without merging**